### ISSUE TITLE
Project does not play preview and could not export

### ISSUE THREAD

**Issue by mickioss** [OpenShot/openshot-qt#3258]
Hello,

I have been editting in OpenShot Editor, but after a while I encountered an error. When opening the project the previewer is not able to play the project or export it in video format, even though the timelines are still intact. If I press play the timeline does play, but nothing is happening. Could someone help me with this problem? I already deleted the .openshot_qt folder and also re-installed the program, but that does not fix the problem. 

**System Details:**
- OpenShot Version: 2.5.0
- Operating System Windows 10

Log:

[openshot-qt.log](https://github.com/OpenShot/openshot-qt/files/4263052/openshot-qt.log)
[libopenshot.log](https://github.com/OpenShot/openshot-qt/files/4263053/libopenshot.log)

**Screenshots:** (Optional)
![image](https://user-images.githubusercontent.com/61557025/75469694-ba5bdf80-598f-11ea-8cee-610bbb9b0e4e.png)

**Comment by mickioss** [comment]
Addition: 
The program is not broken, because if I open a new project it does play imported material

**Comment by SuslikV** [comment]
Can you share your `.osp` file? I think it is broken. And, please update to the latest daily build: https://www.openshot.org/download/#daily because your .mp3 with the diacritics in name may not work in v2.5.0 (release version).

**Comment by mickioss** [comment]
I did update to the latest version (26-02). I shared it in this message!

**Comment by SuslikV** [comment]
Oh my god, I thought that I wouldn't find a solution...
Repaired file:

[IRES1 - repaired04.zip](https://github.com/OpenShot/openshot-qt/files/4264256/IRES1.-.repaired04.zip)

(CRC of the .zip file MD5:AF4B6DD6377B65D0495125FD0EF31AC2
about 8KB to download)

I made assumption that for file `IMG_3358.MOV` both tracks were in use (audio and video). Check the Properties of the clip in OpenShot (should be set to _Auto_). Original data were corrupted here, so its my guess.

--------
Info for the developers:

The log-file has no info about the error because user didn't hit the export button or so.

`.osp` file:
* line 2009, here the both variables should be dictionary type of _Points_ not boolean (its not a _reader_ key) - it causes problems in loading .json during the export, thus dialog wasn't shown:
```
   "has_audio": false,
   "has_video": true,
```
was replaced by:
```
   "has_audio": {
    "Points": [
     {
      "co": {
       "X": 1.0,
       "Y": -1.0
      },
      "interpolation": 2
     }
    ]
   },
   "has_video": {
    "Points": [
     {
      "co": {
       "X": 1.0,
       "Y": -1.0
      },
      "interpolation": 2
     }
    ]
   },
```

* line 8583 the were few useless null objects (it seems to be harmless for v2.5.0 but I removed them)
```
 "4": null,
 "0": null,
 "9": null
```

Next is was the `é` in the filename saved escaped and with the missing starting part of the filename: `Boléro` vs `./u00e9ro` (stored in the file).
